src/test/java/teammates/test/cases/ui/InstructorFeedbackQuestionCopyActionTest.java

```diff
@@ -29,97 +29,97 @@ public void caseSetUp() throws Exception {
         dataBundle = getTypicalDataBundle();
         removeAndRestoreTypicalDataInDatastore();
     }
-    
+
     @Test
-    public void testAccessControl() throws Exception{
+    public void testAccessControl() throws Exception {
         String[] params = new String[]{
                 Const.ParamsNames.FEEDBACK_SESSION_NAME, "First feedback session",
                 Const.ParamsNames.COURSE_ID, "idOfTypicalCourse1"
         };
-        
+
         verifyOnlyInstructorsOfTheSameCourseCanAccess(params);
         verifyUnaccessibleWithoutModifyCoursePrivilege(params);
     }
-    
+
     @Test
-    public void testExecuteAndPostProcess() throws Exception{
-        //TODO: find a way to test status message from session
-        InstructorAttributes instructor1ofCourse1 =
-                dataBundle.instructors.get("instructor1OfCourse1");
-        
+    public void testExecuteAndPostProcess() throws Exception {
+        InstructorAttributes instructor1ofCourse1 = dataBundle.instructors.get("instructor1OfCourse1");
+
         ______TS("Not enough parameters");
-        
+
         gaeSimulation.loginAsInstructor(instructor1ofCourse1.googleId);
         verifyAssumptionFailure();
-        //TODO make sure IFAA does assertNotNull for required parameters then uncomment
-        //verifyAssumptionFailure(Const.ParamsNames.COURSE_ID, instructor1ofCourse1.courseId);
-        
+
+        // This is commented out as InstructorFeedbackAddAction already assertNotNull using Assumption
+        // verifyAssumptionFailure(Const.ParamsNames.COURSE_ID, instructor1ofCourse1.courseId);
+
         ______TS("Typical case");
-        
+
         FeedbackSessionAttributes session1 = dataBundle.feedbackSessions.get("session1InCourse1");
-        FeedbackQuestionAttributes question1 = FeedbackQuestionsLogic.inst().getFeedbackQuestion(session1.feedbackSessionName, session1.courseId, 1);
-        FeedbackQuestionAttributes question2 = FeedbackQuestionsLogic.inst().getFeedbackQuestion(session1.feedbackSessionName, session1.courseId, 2);
-        
+        FeedbackQuestionAttributes question1 = FeedbackQuestionsLogic.
+                                               inst().
```

Comment:
Yupp, we have it as after operator in Teammates Java Style Guide ->

![screen shot 2015-05-21 at 12 07 21 am](https://cloud.githubusercontent.com/assets/6291947/7730507/63ad5c0a-ff4d-11e4-8601-9de98bfc25e7.png)

But there is no explicit convention on whether to have dot operators at the end as in

<code>
FeedbackQuestionsLogic.inst()
          .getFeedbackQuestion(session1.feedbackSessionName, session1.courseId, 1);
</code>
or
<code>
FeedbackQuestion.inst().
          getFeedback...
</code>

so I must have wrongly assumed we are following Google style which is the second one (stated in Google JavaScript style guide which I think we also follow)

Perhaps we should more specifically target dot operators? We have quite a number of chain dots in Teammates.